### PR TITLE
refactor provider types, `includes` not supported on IE

### PR DESCRIPTION
**Pull request by wilson-alcantara**
- [x] remove `includes`; not supported on IE
- [x] direct booking should be `airline` and not facilitated booking

### CODE REVIEW

File: src/flight-search/filtering.js [hunk 1]
```diff
@@ -31,8 +31,17 @@ function filterByProviders(trip, providerFilter) {
 
 function isFareMatchProviderType (fare, providerTypes) {
   if (!providerTypes) return true;
-  var isMatchTypeInstant = (fare.provider.instant && providerTypes.includes('instant'));
-  return isMatchTypeInstant || (providerTypes.includes(fare.provider.type));
+  return _isFacilitatedBooking(fare, providerTypes) || _isDirectBooking(fare, providerTypes);
```
Comment (viettienn) [inline src/flight-search/filtering.js]: looks like if we want to add another provider type in the filter, .e.g `ota`, then we have to update SDK. How we can prevent that?

Comment (wilson-alcantara) [reply src/flight-search/filtering.js]: yes we need to update SDK if we want to add filter for `ota`

Comment (wilson-alcantara) [reply src/flight-search/filtering.js]: what do you mean by `how we can prevent that?`? do you mean that you want to support `ota` even though we aren't using it?

Comment (inyigo) [reply src/flight-search/filtering.js]: i think he means to make providerType filter generic and dynamic, so when new filter comes along no need to change code. not sure if it's possible though

Comment (wilson-alcantara) [reply src/flight-search/filtering.js]: Oh I see, yup it's possible

**Comment by wilson-alcantara** [comment]
@viettienn @inyigo updated.

**Pull request merged**